js/components/DeploymentComponent.jsx

```diff
@@ -1,13 +1,15 @@
 /** @jsx React.DOM */
 
+var lazy = require("lazy.js");
```

Comment:
Yes, as this is a capital L in the docs of lazy.js too, this makes sense.
Indeed it doesn't follow the guidelines of JavaScript: The Good Parts, but fair enough.. :)